### PR TITLE
dug --json

### LINKED ISSUE #106: JSON
https://www.rfc-editor.org/rfc/rfc8427.html

### PR DESCRIPTION
**Pull request by kazu-yamamoto**
Should fix #106.

### CODE REVIEW

File: dnsext-full-resolver/dug/JSON.hs [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+        , ("TYPE", pretty $ fromTYPE rrtype)
+        , ("TYPEname", string rrtype)
+        , ("TTL", pretty ttl)
+        , rd rrtype rdata
+        ]
+
+{- FOURMOLU_DISABLE -}
+rd :: TYPE -> RData -> (String, Doc ann)
+rd A     v = ("rdA",         justQuote (fromRData v :: Maybe RD_A))
+rd AAAA  v = ("rdAAAA",      justQuote (fromRData v :: Maybe RD_AAAA))
```
Comment (khibino) [inline dnsext-full-resolver/dug/JSON.hs:80]:
```suggestion
rd A     v = ("rdataA",      justQuote (fromRData v :: Maybe RD_A))
rd AAAA  v = ("rdataAAAA",   justQuote (fromRData v :: Maybe RD_AAAA))
```

adjust field members to https://datatracker.ietf.org/doc/html/rfc8427#section-2.3 ?

Comment (kazu-yamamoto) [reply dnsext-full-resolver/dug/JSON.hs]: [5aab4f4](https://github.com/kazu-yamamoto/dnsext/pull/115/commits/5aab4f459ade2282a521e585a704b429413d503f) fixes them

**Review by khibino — APPROVED** [review]
LGTM

**Comment by kazu-yamamoto** [comment]
merged.

**Pull request merged**